.github/workflows/test.yml

```
@@ -29,6 +29,13 @@ jobs:
         run: pnpm test
         env:
           FORCE_COLOR: 1
+      - name: Upload artifacts
```

```suggestion
      - name: Upload web assets
```

Artifact is abstract term. We can be more precise.